tests/spread/rockcraft/plugin-ant/task.yaml

```diff
@@ -0,0 +1,17 @@
+summary: Maven plugin tests
+execute: |
+  SCENARIO="plugin-ant"
+  ROCK_FILE="${SCENARIO}_0.1_amd64.rock"
+  IMAGE="${SCENARIO}:0.1"
+
+  # Build the rock & load it into docker
+  run_rockcraft pack
+  echo expecting ${ROCK_FILE}
+  test -f ${ROCK_FILE}
+  rockcraft.skopeo copy --insecure-policy  oci-archive:${ROCK_FILE} docker-daemon:${IMAGE}
+  rm ${ROCK_FILE}
+
+  # Run the packaged project
+  docker run --rm $IMAGE exec /usr/lib/jvm/java-21-openjdk-amd64/bin/java -jar /jar/HelloWorld.jar | MATCH "hello world"
+
+  docker system prune -a -f
```

Comment:
no need to do this, it's [already done](https://github.com/canonical/rockcraft/blob/main/spread.yaml#L96) for each test